app/assets/stylesheets/base/_games.css.scss

```diff
@@ -35,3 +35,6 @@
   height: 100%;
   vertical-align: middle;
 }
+#dialog-message {

```

Comment:
i believe bootstrap has a `.hidden` class you can use that does exactly this.

http://getbootstrap.com/css/#helper-classes-show-hide